packages/assets-controllers/src/assetsUtil.ts

```diff
@@ -292,3 +293,68 @@ export async function reduceInBatchesSerially<
   const finalResult = workingResult as Result;
   return finalResult;
 }
+
+/**
+ * Maps an OpenSea V2 NFT to the V1 schema.
+ * Should work for both 'get_nft' and 'list_nfts_by_account' responses.
+ * @param nft - The V2 NFT to map.
+ * @returns The NFT in the V1 schema.
+ */
+export function mapOpenSeaNftV2ToV1(nft: any): ApiNft {
+  return {
+    token_id: nft.identifier,
+    num_sales: null,
+    background_color: null,
+    image_url: nft.image_url,
+    image_preview_url: nft.image_url,
+    image_thumbnail_url: nft.image_url,
+    image_original_url: null,
+    animation_url: nft.animation_url,
```

Comment:
I checked the [OpenAPI spec](https://docs.opensea.io/reference/openapi-definition) for both `${OPENSEA_PROXY_URL}/chain/${chain}/contract/${contractAddress}/nfts/${tokenId}` and `${OPENSEA_PROXY_URL}/chain/${chain}/account/${address}/nfts` and it seems like both return NftModel rather than DetailedNftModel, so it seems like this wouldn't be available. Perhaps we should just set this to `null`?

Reply:
Ah, strange. Okay!